coriolis/providers/replicator.py

```diff
@@ -390,12 +362,9 @@ def _parse_replicator_conn_info(self, conn_info):
             "port": CONF.replicator.port,
         }
 
+    @utils.retry_on_error()
     def _copy_file(self, ssh, localPath, remotePath):
-        tmp = tempfile.mkstemp()[1]
-        try:
-            os.remove(tmp)
-        except BaseException:
-            pass
+        tmp = os.path.join("/tmp", str(uuid.uuid4()))
```

Comment:
Any reason why we'd not want to use `tempfile.mkstemp` given that the dir is on the local machine?

Reply:
The file is remote. tempfile creates a local file. No use in creating a local file just to get a name we could use remotely.

Reply:
My bad, I fleetingly saw the separate `remotePath` and assumed this one would be a local one.